internal/builtin/providers/terraform/provider.go

```diff
@@ -33,7 +33,7 @@ func (p *Provider) GetProviderSchema() providers.GetProviderSchemaResponse {
 		ResourceTypes: map[string]providers.Schema{
 			"terraform_data": dataStoreResourceSchema(),
 		},
-		EphemeralResourceTypes: map[string]providers.Schema{},
+		EphemeralTypes: map[string]providers.Schema{},
```

Comment:
Was there a particular motivation behind this rename? It feels to me like we're introducing a third convention here.

I mean, we could have:

1. "data sources" + "resources" + "ephemeral resources"
2. "data source types" + "resource types" + "ephemeral resource types"
3. "data types" + "resource types" + "ephemeral types"

That is to say the current state is _already_ inconsistent but I'm not sure this new name is helping.

Reply:
It wasn't me 😆 I was just following previously published code.